receiver/prometheusreceiver/internal/metricsbuilder.go

```diff
@@ -41,27 +42,30 @@ var errEmptyBoundaryLabel = errors.New("BucketLabel or QuantileLabel is empty")
 var dummyMetrics = make([]*metricspb.Metric, 0)
 
 type metricBuilder struct {
-	hasData           bool
-	hasInternalMetric bool
-	mc                MetadataCache
-	metrics           []*metricspb.Metric
-	numTimeseries     int
-	droppedTimeseries int
-	logger            *zap.SugaredLogger
-	currentMf         MetricFamily
+	hasData            bool
+	hasInternalMetric  bool
+	mc                 MetadataCache
+	metrics            []*metricspb.Metric
+	numTimeseries      int
+	droppedTimeseries  int
+	useStartTimeMetric bool
+	startTime          float64
+	logger             *zap.SugaredLogger
```

Comment:
Q.: this is not a change but typically we use `*zap.Logger`. Are you aware of any specific reason to use the `SugaredLogger`? The former is strong typed and as result a bit more verbose on the source but the rest of the code base already uses it....

Reply:
I'm not sure why the original author used SugaredLogger but I don't see any particular need for it - I can write a follow-up pr to remove it.